src/components/exchange/CurrencySelectionList.js

```diff
@@ -64,10 +65,8 @@ const CurrencySelectionList = (
     >
       {showList && !showSkeleton && (
         <Centered flex={1}>
-          {showNoResults ? (
-            loading ? null : (
-              <NoCurrencyResults />
-            )
+          {showGhost ? (
```

Comment:
renamed "showNoResults" to "showGhost" bc it was a bit confusing bc the ghost is shown not just with no results showing